src/modlunky2/ui/trackers/category.py

```diff
@@ -33,19 +34,24 @@ def __init__(self, parent, modlunky_config: Config, *args, **kwargs):
             Image.open(ICON_PATH / "cat2.png").resize((24, 24), Image.ANTIALIAS)
         )
 
+        current_row = 0
+
         self.category_button = ttk.Button(
             self,
             image=self.cat_icon,
             text="Category",
             compound="left",
             command=self.launch,
         )
-        self.category_button.grid(row=0, column=0, pady=5, padx=5, sticky="nswe")
+        self.category_button.grid(
+            row=current_row, column=0, pady=5, padx=5, sticky="nswe"
+        )
 
         self.always_show_modifiers = tk.BooleanVar()
         self.always_show_modifiers.set(
             modlunky_config.trackers.category.always_show_modifiers
         )
+
         self.always_show_modifiers_checkbox = ttk.Checkbutton(
```

Comment:
I think the UI part is pretty minor, but if you want to make this more like my sketch...

* Create a subclass of `ttk.LabelFrame` to hold the "Modifiers" checkboxes, and use it here (instead of directly adding them)
* Put a `ttk.Separator` between the "Always Show" checkbox and the per-label ones